Assets/Scripts/Game/Items/Interactable.cs

```diff
@@ -15,13 +15,16 @@ void Start()
         {
             tooltip = Instantiate(tooltipPrefab, transform.position, Quaternion.identity, HUD.TooltipParent);
             tooltip.SetTarget(transform);
-            tooltip.SetText(interactHint);
+            tooltip.SetText("[E] " + interactHint);
```

Comment:
Are there plans to make this work with controllers and/or rebound keys?

Reply:
Yes, definitely! After the core milestone, we'll have a task for setting up proper keybind customization. Until then, we'll just build for a working PC demo.